roles/ceph-nfs/tasks/docker/copy_configs.yml

```diff
@@ -1,10 +1,28 @@
 ---
-- name: push ceph files to the ansible server
-  fetch:
-    src: "{{ item.0 }}"
-    dest: "{{ fetch_directory }}/docker_mon_files/{{ item.0 }}"
-    flat: yes
+- name: set config and keys paths
+  set_fact:
+    ceph_config_keys:
+      - /etc/ceph/ceph.conf
+      - /etc/ceph/ceph.client.admin.keyring
+      - /var/lib/ceph/radosgw/keyring
+
+- name: stat for config and keys
+  local_action: stat path={{ fetch_directory }}/docker_mon_files/{{ item }}
+  with_items: ceph_config_keys
+  changed_when: false
+  become: false
+  failed_when: false
+  register: statconfig
+
+- name: try to fetch config and keys
+  copy:
+    src: "{{ fetch_directory }}/docker_mon_files/{{ item.0 }}"
+    dest: "{{ item.0 }}"
+    owner: "64045"
+    group: "64045"
+    mode: 0644
+  changed_when: false
   with_together:
     - ceph_config_keys
     - statconfig.results
-  when: item.1.stat.exists == false
+  when: item.1.stat.exists == true
```

Comment:
We need to copy all the files that aren't there yet.  copy_configs.yaml wasn't included before (it was a straight copy of the RGW file), so the fact that it was wrong before didn't matter.